File: help/eng_gb/exercicios_da_ficha_4/Ficha04_004/Ficha04_004.js
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Number \t: " );// print text
    num = parseInt(prompt("Number \t: " ,"0") );
    
    document.write(num + "\n");// print input 
    if(num > 0 ){
        document.write("Positive" );
    }else{
        if(num == 0 ){
            document.write("Null" );
        }else{
            document.write("Negative" );
        }
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:27

File: help/eng_gb/exercicios_da_ficha_4/Ficha04_004/Ficha04_004.cpp
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha04_004 {

    public:
        int main() {
            cout << "Number \t: ";
            int num;
            cin >> num;
            if(num > 0 ){
                cout << "Positive";
            }else{
                if(num == 0 ){
                    cout << "Null";
                }else{
                    cout << "Negative";
                }
            }
            return 0;
        }
};// end of class Ficha04_004
// Execute main function
int main(){
    Ficha04_004 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	domingo, 2022 set 18 23:27
